### PR TITLE
Remove all references to a specific element type.

### PR DESCRIPTION
**Pull request by AAJELLAL**

### CODE REVIEW

File: src/main/java/org/gridsuite/directory/server/NotificationService.java [hunk 1]
```diff
@@ -35,7 +35,8 @@ public class NotificationService {
     public static final String HEADER_NOTIFICATION_TYPE = "notificationType";
     public static final String HEADER_ELEMENT_NAME = "elementName";
     public static final String HEADER_STUDY_UUID = "studyUuid";
```
Comment (sBouzols) [inline src/main/java/org/gridsuite/directory/server/NotificationService.java:37]: remove HEADER_STUDY_UUID as well

File: src/main/java/org/gridsuite/directory/server/DirectoryService.java [hunk 2]
```diff
@@ -52,8 +50,6 @@ public class DirectoryService {
     private static final String CATEGORY_BROKER_INPUT = DirectoryService.class.getName() + ".input-broker-messages";
     private static final Logger LOGGER = LoggerFactory.getLogger(DirectoryService.class);
 
-    private final StudyService studyService;
```
Comment (sBouzols) [inline src/main/java/org/gridsuite/directory/server/DirectoryService.java:55]: Then remove StudyService.java file as well ?

**Review by sBouzols — CHANGES_REQUESTED** [review]
There are still occurences of STUDY, CONTINGENCY_LIST, FILTER, MODIFICATION, CASE etc...

**Comment by AAJELLAL** [comment]
> There are still occurences of STUDY, CONTINGENCY_LIST, FILTER, MODIFICATION, CASE etc... Why do we have a `consumeStudyUpdate` here...



> There are still occurences of STUDY, CONTINGENCY_LIST, FILTER, MODIFICATION, CASE etc... Why do we have a `consumeStudyUpdate` here...

I have cleaned everything except the two test methods (testCountUserCases and testCountCasesByUser), which will be done later in a separate user story.

File: src/main/java/org/gridsuite/directory/server/DirectoryService.java (hunk 2 above)
Comment (AAJELLAL) [reply src/main/java/org/gridsuite/directory/server/DirectoryService.java:55]: Done.

**Pull request merged**